common/model/books.ts

```diff
@@ -0,0 +1,25 @@
+// @flow
+import type { GenericContentFields } from './generic-content-fields';
+import type { HTMLString } from '../services/prismic/types';
+import type { ImageType } from './image';
+
+type Review = {
+  text: HTMLString,
+  citation: HTMLString,
+};
+
+export type Book = GenericContentFields & {
+  type: 'books',
+  subtitle: string | null,
+  orderLink: string | null,
+  price: string | null,
+  format: string | null,
+  extent: string | null,
+  isbn: string | null,
+  reviews: (Review[]) | null,
+  datePublished: Date | null,
+  authorName: string | null, // This is structuredText in Prismic >.<
+  authorImage: string | null, // This is a link in Prismic >.<
```

Comment:
Why do we have it as a string?

Reply:
I'm guessing because `// This is a link in Prismic >.<`